collections/harvestparams.php

```diff
@@ -294,7 +294,7 @@
 					<label for="hascoords"><?php echo isset($LANG['HAS_COORDS'])?$LANG['HAS_COORDS']:'Limit to Specimens with Geocoordinates Only'; ?></label>
 				</div>
 				<div>
-					<input type='checkbox' name='includecult' id='includecult' value='1' />
+					<input type='checkbox' name='includecult' id='includecult' value='<?php echo $CSS_VERSION_RELEASE === 'uswds' ? '1' : '0' ?>' <?php echo $CSS_VERSION_RELEASE === 'uswds' ? 'checked' : '' ?> />
```

Comment:
Wom wom shouldn't have just changed the checked attribute. Should also have changed the value.